server.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+from config import settings
+from utils import ModelName, selectModel, send_rest_api_req
+
+class Prompt(BaseModel):
+    question: str
+    context: str
+    model: ModelName = ModelName.distilbert
+
+app = FastAPI()
+
+@app.get("/")
+def home_route():
+    return {"Message": "welcome to bons.ai"}
+
+@app.post("/prompt")
+async def prompt_llm(prompt: Prompt):
+    context = prompt.context
+    question = prompt.question
+    headers = {"Authorization": f"Bearer {settings.api_token}"}
+    payload = {
+            "inputs": {
+                "question": f"{question}",
+                "context": f"{context}",
+            }
+        }
+    api_url = selectModel(prompt.model)
```

Comment:
You are missing validation in `selectModel` about whether the input from the user is valid or not.

Reply:
the validation for the request data received from client is being handled by pydantic package. but I can add separate validation in selectModel api.